From: Ma Wupeng <mawupeng1@huawei.com>
Subject: doc: update params of memhp_default_state=
Date: Wed, 2 Aug 2023 15:43:12 +0800

Commit 5f47adf762b7 ("mm/memory_hotplug: allow to specify a default
online_type") allows to specify a default online_type which make online
memory to kernel or movable zone possible but fail to update to doc. 
Update doc to fit this change.

Link: https://lkml.kernel.org/r/20230802074312.2111074-1-mawupeng1@huawei.com
Signed-off-by: Ma Wupeng <mawupeng1@huawei.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Cc: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/admin-guide/kernel-parameters.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/admin-guide/kernel-parameters.txt~doc-update-params-of-memhp_default_state=
+++ a/Documentation/admin-guide/kernel-parameters.txt
@@ -3105,7 +3105,7 @@
 			[KNL,SH] Allow user to override the default size for
 			per-device physically contiguous DMA buffers.
 
-	memhp_default_state=online/offline
+	memhp_default_state=online/offline/online_kernel/online_movable
 			[KNL] Set the initial state for the memory hotplug
 			onlining policy. If not specified, the default value is
 			set according to the
_
